src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -125,21 +123,33 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
 
     return app ? (
       <EuiFlexItem className="kbnOverviewApps__item" key={appId}>
-        <RedirectAppLinks application={application}>
-          <EuiCard
-            description={app?.subtitle || ''}
-            href={addBasePath(app.path)}
-            onClick={() => {
-              trackUiMetric(METRIC_TYPE.CLICK, `app_card_${appId}`);
-            }}
-            image={addBasePath(
-              `/plugins/${PLUGIN_ID}/assets/kibana_${appId}_${IS_DARK_THEME ? 'dark' : 'light'}.svg`
-            )}
-            title={app.title}
-            titleElement="h3"
-            titleSize="s"
-          />
-        </RedirectAppLinks>
+        <RedirectAppLinksKibanaProvider
+          coreStart={{
+            application: {
+              currentAppId$: application.currentAppId$,
+              navigateToUrl: application.navigateToUrl,
+            },
+          }}
+          {...application}
+        >
+          <RedirectAppLinks>
+            <EuiCard
+              description={app?.subtitle || ''}
+              href={addBasePath(app.path)}
+              onClick={() => {
+                trackUiMetric(METRIC_TYPE.CLICK, `app_card_${appId}`);
+              }}
+              image={addBasePath(
+                `/plugins/${PLUGIN_ID}/assets/kibana_${appId}_${
```

Comment:
Curious ton know how we are testing this to make sure that a test fails if the svg files are removed or renamed. Do you know?

It seems a good candidate for component integration tests.

Reply:
I agree we need integration tests for this plugin, will add them in the next interation.